html/Index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8" />
+<title>Portforlio-Home</title>
+<meta name="viewport" content="width=device-width,initial-scale=1.0" />
+<link rel="stylesheet" href="../css/style.css" />
+</head>
+<body>
+
+<div class="container">
+<header class="header">
+<img src="../Images/geometry.png" alt="">
+<img src="../Images/ic_menu.png" class="icon" alt="">
+</header>
+
+<article class="content">
+  <header>
+    <h1>Iyadi  <br>Osita Cyril</h1>   
+  </header>
+  <p>Hi there!, I love software development, it can be very tasking,
+    but there is nothing I'd rather do. I can help you bring products,
+    features and websites from your imagination into reality. 
+    Browse my works, if you like them feel free to contact me, 
+    I'm happy to work with you.
+  </p> 
+  <button class="collaborationbutton" type="button"><span class="collaboration">Start collaboration</span></button>
+</article>
```

Comment:
The container of the section is a `section` tag, please, add your content in `<section></section>`.

- [ ] Put your container in a `section` tag.